src/lib/apiTypesV2/Entity.cpp

```diff
@@ -57,37 +57,37 @@ Entity::~Entity()
 */
 std::string Entity::render(ConnectionInfo* ciP, RequestType requestType, bool comma)
 {
-
```

Comment:
Sorry, I didn't get the comment... GH doesn't render new comments in real-time...